### PR TITLE
Remove TestUtil disposable dir function and copy instead of moving na…

### PR DESCRIPTION
**Pull request by BrennanConroy**
…tive assemblies

@moozzyk @davidfowl

### CODE REVIEW

**Comment by davidfowl** [comment]
We need to see why Travis is failing (and only one of them)

**Comment by BrennanConroy** [comment]
"The remote server returned an error: (404) Not Found." when trying to download dnxcoreclr

**Comment by davidfowl** [comment]
Forcing a rerun

**Comment by davidfowl** [comment]
:shipit:

File: makefile.shade [hunk 1]
```diff
@@ -630,8 +630,10 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(VS_ONECORE_ROOT) && Directory.GetFil
                         var sourceFile = Path.Combine(target.TargetFolder, native + nativeExtension);
                         if (File.Exists(sourceFile))
                         {
-                            Log.Info("Moving " + native + " to " + targetFilePath);
-                            File.Move(Path.Combine(target.TargetFolder, native + nativeExtension), targetFilePath);
+                            Log.Info("Copying " + sourceFile + " to " + targetFilePath);
+                            File.Copy(sourceFile, targetFilePath);
+                        } else {
```
Comment (moozzyk) [inline makefile.shade]: formatting

**Comment by moozzyk** [comment]
:ship::it:

File: makefile.shade (hunk 1 above)
Comment (BrennanConroy) [inline makefile.shade]: Whoops, old habits

**Comment by BrennanConroy** [comment]
2fc505893d82def50e0828b8a7030ff54ee3b637

**Pull request closed without merging**